src/js/_store/modules/categories.js

```diff
@@ -92,10 +112,29 @@ export default {
           if (err.data) {
             commit('failRequest', { message: err.response.data.message });
           } else {
-            commit('failRequest', { message: 'ネットに接続がされていないかサーバーとの接続がされていません。ご確認ください。' });
+            commit('failRequest', { message: err.message });
           }
         });
       });
     },
+    deleteCategory({ commit, rootGetters }) {
+      return new Promise((resolve, reject) => {
+        commit('clearMessage');
+        const data = new URLSearchParams();
+        data.append('id', rootGetters['categories/deleteCategory']);
+        axios(rootGetters['auth/token'])({
+          method: 'DELETE',
+          url: `category/${rootGetters['categories/deleteCategory']}`,
+          data,
+        }).then(() => {
+          commit('doneDeleteCategory');
+          commit('displayDoneMessage', { message: 'カテゴリーを削除しました！' });
```

Comment:
displayDoneMessageという関数をpostCategory（）にも使いまわせると思うんで使ってみましょう。
```suggestion
          commit('displayDoneMessage', { message: '新規カテゴリの追加が完了しました！' });
```

Reply:
be4e699fa6253381e933c856429cd15e9746e15b
こちら、使いまわしたのと、doneDisplayMessageに名称変更いたしました！他のものがdoneから始まっていたため！